site/content/en/docs/tasks/run_plain_pods.md

```diff
@@ -40,12 +40,26 @@ This guide is for [batch users](/docs/tasks#batch-user) that have a basic unders
            values: [ "true", "True", "yes" ]
    ```
 
-2. Pods that belong to other API resources managed by Kueue are excluded from being queued by `pod` integration. 
+2. Kueue will run webhooks for all created pods if pod integration is enabled. Webhook namespaceSelector could be 
+   used to filter the pods to reconcile. Default webhook namespaceSelector is:
+   ```yaml
+   matchExpressions:
+   - key: kubernetes.io/metadata.name
+     operator: NotIn
+     values: [ kube-system, kueue-system ]
+   ```
+   
+   In case of [installation via Helm](/docs/installation/#install-via-helm), the webhook namespace selector 
+   will match the `integrations.podOptions.namespaceSelector` in the `values.yaml`.
+   
+   Please keep in mind that namespaceSelector should never match the kueue namespace.
```

Comment:
```suggestion
   Make sure that namespaceSelector never matches the kueue namespace, otherwise the Kueue deployment won't be able to create Pods.
```